src/accordian/Accordion.tsx

```diff
@@ -0,0 +1,197 @@
+    content: accordionContent[];
+    accordionMultiPanel?: boolean;
+};
+
+type accordionContent = {
+    title: string;
+    itemComponent: React.ReactNode;
+};
+
+type SignPostState = {
+    panelItems: any;
+    prevItemIndex: number;
+};
+
+export class Accordion extends React.Component<AccordianProps> {
+    state: SignPostState = {
+        prevItemIndex: -1,
+        panelItems: [],
+    };
+
+    componentDidMount() {
+        this.getAccordionContent();
+    }
+    handleButtonClick = (index: any, accordionMultiPanel: any) => {
+        let items = this.state.panelItems;
+        if (this.state.prevItemIndex != -1 && this.state.prevItemIndex === index) {
+            items[index].isOpen = false;
+            this.setState({prevItemIndex: -1});
+            items[this.state.prevItemIndex].content = this.getItemContent(
+                this.state.prevItemIndex,
+                items[this.state.prevItemIndex].title,
+                true,
+                accordionMultiPanel,
+            );
+            this.setState({panelItems: items});
+        } else {
+            if (!accordionMultiPanel && this.state.prevItemIndex != -1) {
+                items[this.state.prevItemIndex].isOpen = false;
+                items[this.state.prevItemIndex].content = this.getItemContent(
+                    this.state.prevItemIndex,
+                    items[this.state.prevItemIndex].title,
+                    true,
+                    accordionMultiPanel,
+                );
+            }
+            items[index].isOpen = true;
+            items[index].content = this.getItemContent(index, items[index].title, false, accordionMultiPanel);
+            this.setState({prevItemIndex: index});
+            this.setState({panelItems: items});
+        }
+        this.accordionPanel();
+    };
+
+    private accordionPanel(): React.ReactElement {
+        return (
+            <div>
+                {this.state.panelItems.map((content: any) => {
+                    return (
+                        <div className={classNames([ClassNames.ACCORDION_PANEL])}>
+                            {content.content}
+                            {content.isOpen && content.isOpen === true ? (
+                                this.accordionContent(content)
+                            ) : (
+                                <div
+                                    className={classNames([ClassNames.ACCORDION_COLLAPSED_CONTENT])}
+                                    aria-hidden="true"
+                                />
+                            )}
+                        </div>
+                    );
+                })}
+            </div>
+        );
+    }
+
+    private accordionContent(content: any): React.ReactElement {
+        return (
+            <div className={classNames([ClassNames.ACCORDION_COLLAPSED_CONTENT])} aria-hidden="false">
+                <div className={classNames([ClassNames.ACCORDION_TOGGLE_CONTENT])} style={{display: "block"}}>

```

Comment:
`ACCORDION_TOGGLE_CONTENT` is not in `ClassNames.tsx`

Reply:
addressed